photo/mutations.py

```diff
@@ -63,16 +63,19 @@ def create_picture(self, input: PictureInput, picture: Upload) -> PictureType:
         image_bytes = BytesIO()
         picture.save(image_bytes, format="JPEG")
         image_bytes.seek(0)
-        image_file = SimpleUploadedFile(
-            "sample.jpg", image_bytes.getvalue(), content_type="image/jpeg"
-        )
 
-        user = User.objects.get(email=input.user)
+        user = User.objects.get(id=input.user)
+
+        picture_object = Picture(user=user)
+        picture_object.save()
 
-        new_picture = Picture(user=user, file=image_file)
-        new_picture.save()
+        image_file = SimpleUploadedFile(
+            str(picture_object.id), image_bytes.getvalue(), content_type="image/jpeg"
```

Comment:
what happens if someone uploads a PNG?

Reply:
Changed the format of the pictures to webp